### DISCUSSION TITLE
hr Livestream bei Sportübertragungen nicht via Zapp

### DISCUSSION THREAD

**Discussion by ghost** [mediathekview/zapp#305] [Q&A] [answered]
Vielleicht habe ich ja einen falschen Denkansatz bzgl der Nutzung von Zapp für Livestreams. Seit Monaten frage ich mich, warum ich beispielsweise an Sonntagen mit Live-Sportübertragungen auf hr den Livestream grundsätzlich nicht mit Zapp schauen kann. Das betrifft u. a. Radrennen und Ironman-Wettkämpfe, die von hr live übertragen werden.
Auf der Website des hr kann ich den Livestream im Browser des Androiden ebenso korrekt sehen wie in der App ARD Mediathek. In Zapp sehe ich dann stundenlang Helikopterflüge über die Frankfurter Skyline mit dem bekannten Hinweis "Aus rechtlichen Gründen können wir die aktuelle Sendung nicht als Livestream anbieten."

Nutzt Zapp für den Livestream einen anderen URL?

**Comment by cemrich** [comment, marked as answer]
Das scheint so. Meistens gibt es einen Stream mit Geoblocking, auf dem dann aber mehr Inhalte zu sehen sind und einen Stream ohne, der dann aber nicht alle Inhalte zeigt. Man kann darüber diskutieren, welche Version sinnvoller ist - aber bis jetzt ist mir das nicht einmal aufgefallen. Ich werde das mal im Auge behalten.
